docs/howto/uploading_smart_contract.md

```diff
@@ -48,12 +49,16 @@ two nodes and upload a smart contract via the circuit.
     <circuit-id> tutorial   alpha-node-000;beta-node-000
     ```
 
+    ```
+    $ export CIRCUIT_ID=<circuit_id>
```

Comment:
Suggest we make this an example without the shell characters, maybe with some preceeding text that says "For the following commands, we set the CIRCUIT_ID environment variable; this environment variable is for the purposes of this procedure and is not used directly by the splinter CLI commands.

Set CIRCUIT_ID based on the output of the proposals subcommand; for example:"